SendWelcomeMessagesActivity/index.ts

```diff
@@ -116,20 +121,35 @@ Per approfondimenti ti invitiamo a consultare la sezione [servizi](ioit://SERVIC
     })
 ];
 
+const SERVICES_API_BASE_HEADERS = {
+  "Content-Type": "application/json"
+};
+
 /**
- * Send a single welcome message using the
- * Digital Citizenship Notification API (REST).
+ * Send a single welcome message using the Services API.
+ *
+ * If the apiKey is empty, the request will include the authorization
+ * headers that the Services API expects from the API Manager.
  */
 async function sendWelcomeMessage(
   url: string,
-  apiKey: string,
+  apiKey: Option<NonEmptyString>,
```

Comment:
I'm not super enthusiast of taking option as argument, this is a choice that should be made by the caller (same scope of maybeApiKey) and early bind apiKey ie something like:

```ts
const sendWelcomeMessage = (() => 
maybeApiKey.fold(
  () => sendWelcomeMessageInternal() // returns a function (url, message) => Promise<number>
  apiKey => sendWelcomeMessageApim(apiKey) // returns a function (url, message) => Promise<number>
))()
```